config/convert_deepen_to_t4_segmetation_painting_sample.yaml

```
@@ -0,0 +1,31 @@
+task: convert_deepen_to_t4
+description:
+  visibility:
+    full: "No occlusion of the object."
+    most: "Object is occluded, but by less than 50%."
+    partial: "The object is occluded by more than 50% (but not completely)."
+    none: "The object is 90-100% occluded and no points/pixels are visible in the label."
+  camera_index:
+    CAM_BACK_RIGHT: 1
+    CAM_BACK: 2
+    CAM_BACK_LEFT: 3
+    CAM_FRONT: 4
+    CAM_FRONT_LEFT: 5
+    CAM_FRONT_NARROW: 6
+    CAM_FRONT_RIGHT: 7
+    CAM_FRONT_WIDE: 8
```

```suggestion
    CAM_BACK_RIGHT: 0
    CAM_BACK: 1
    CAM_BACK_LEFT: 2
    CAM_FRONT: 3
    CAM_FRONT_LEFT: 4
    CAM_FRONT_NARROW: 5
    CAM_FRONT_RIGHT: 6
    CAM_FRONT_WIDE: 7
```

Camera id in t4dataset starts from `0`. cf. https://github.com/tier4/tier4_perception_dataset/blob/f79e18d4f9958f854ec343f369c88ce6ec358854/config/convert_deepen_to_t4_tlr_sample.yaml#L9-L10